certify-service/src/test/resources/data.sql

```diff
@@ -0,0 +1,8 @@
+--INSERT INTO KEY_POLICY_DEF(APP_ID,KEY_VALIDITY_DURATION,PRE_EXPIRE_DAYS,ACCESS_ALLOWED,IS_ACTIVE,CR_BY,CR_DTIMES) VALUES ('ROOT', 1095, 50, 'NA', true, 'mosipadmin', now());
+--INSERT INTO KEY_POLICY_DEF(APP_ID,KEY_VALIDITY_DURATION,PRE_EXPIRE_DAYS,ACCESS_ALLOWED,IS_ACTIVE,CR_BY,CR_DTIMES) VALUES('IDP_SERVICE', 1095, 50, 'NA', true, 'mosipadmin', now());
+--INSERT INTO KEY_POLICY_DEF(APP_ID,KEY_VALIDITY_DURATION,PRE_EXPIRE_DAYS,ACCESS_ALLOWED,IS_ACTIVE,CR_BY,CR_DTIMES) VALUES('IDP_PARTNER', 1095, 50, 'NA', true, 'mosipadmin', now());
+
```

Comment:
Remove the unwanted commented items and remove the esignet specific client details also.

Reply:
removed